csharp/Arrow/ArrowTimeUnit.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+
+namespace ParquetSharp.Arrow
+{
+    /// <summary>
+    /// Arrow TimeUnit enum, with values corresponding to those used in the C++ library.
+    /// These do match the C# Arrow library enum values but there's no guarantee that will always be true.
```

Comment:
Do I understand this correctly? If we had a guarantee that values in the C# Arrow library and C++ library match, we wouldn't have to implement the `ArrowTimeUnit`? Maybe there is no guarantee, but it would be madness if C++ and C# libraries defined different enum values, so I find this an overkill. (But I might be wrong, so feel free to ignore my opinion).

Reply:
Yeah, you're right, I was probably overthinking this. I will simplify this and remove the extra enum. I think it might be sensible to have a unit test to verify the enum values match though